pixorama_server/docker-compose.yaml

```diff
@@ -8,14 +8,14 @@ services:
     environment:
       POSTGRES_USER: postgres
       POSTGRES_DB: pixorama
-      POSTGRES_PASSWORD: "PASSWORD"
```

Comment:
Same here, let's keep the old config or it will not run locally.

Reply:
Resolved